docs/source/examples.rst

```diff
@@ -108,6 +108,16 @@ You will find below an example Jupyter notebook showcasing the usage of the Deep
 
    examples/10-DeepTCN-examples.ipynb
 
+Temporal Fusion Transformer (TFT) Model
+===================
+
+You will find below an example Jupyter notebook showcasing the usage of the TFT model
```

Comment:
Actually could we make those lines less verbose? No need to say each time "You will find below an example ..."

We could just say
```
TFT model example notebook:
```

And similarly for all notebooks.